Resources/config/yui.yml

```diff
@@ -653,6 +653,9 @@ system:
                 ez-expandable:
                     requires: ['view']
                     path: %ez_platformui.public_dir%/js/extensions/ez-expandable.js
+                ez-height-fit:
+                    requires: ['view']
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/220/files#r29037735 (node-screen and node-style)